### ISSUE TITLE
Starting from Salesforce Winter 23 testing community sites fails in chromium, works in firefox

### ISSUE THREAD

**Issue by alexanderg1982** [cypress-io/cypress#24303]
### Current behavior

Starting Winter 23 release experience cloud sites not loading correctly in chromium based browsers (electron 106, chrome 105, edge 105), working correctly in firefox 105.
The simple page is https://alexdebug-moj.cs88.force.com/s/mytest which includes simple html "Use this HTML Editor to add your own markup."

### Desired behavior

Load correctly the page

### Test code to reproduce

https://alexdebug-moj.cs88.force.com/s/mytest

### Cypress Version

10.10.0

### Node version

v16.15.1

### Operating System

Edition - Windows 10 Enterprise | Version - 21H1 | OS build	- 19043.2130 | Experience - Windows Feature Experience Pack 120.2212.4180.0

### Debug Logs

```shell
https://github.com/alexanderg1982/cypress-test-tiny-err-example1/blob/master/failing_debug_electron.txt
https://github.com/alexanderg1982/cypress-test-tiny-err-example1/blob/master/passing_firefox.txt
```

**Comment by AtofStryker** [comment]
Hi @alexanderg1982. Thank you for opening an issue. Does this issue sound similar to the one described in #23958? I was able to reproduce the error in chrome, but not in firefox as you described? Added you test case to the https://github.com/AtofStryker/cypress-issue-23958 repo as it looks related

<img width="1672" alt="Screen Shot 2022-10-20 at 10 10 02 AM" src="https://user-images.githubusercontent.com/3980464/196972514-7dfd2dbd-7371-4e8e-b840-b718c04092ab.png">


in chrome: 

<img width="1344" alt="Screen Shot 2022-10-20 at 10 08 14 AM" src="https://user-images.githubusercontent.com/3980464/196972939-b5e84c86-d077-4616-a790-d75e5947f9ef.png">
